ui/translations/en-us.yaml

```diff
@@ -21,6 +21,16 @@ page:
     title: 'Builds'
   deployments:
     title: 'Deployments'
+    destroyed_label: 'Destroyed'
+    external_url_label: 'Visit Deployment'
+    replaced_label: 'Replaced by '
+    empty_state: 'There are no deployments to display for this app yet'
+    destroyed_and_hidden_state: 'Some deployments have been destroyed and are hidden.'
```

Comment:
I'd say remove the trailing period from this sentence as we typically don't end single sentences with a period in the UI